packages/dataviews/src/search-widget.tsx

```diff
@@ -39,8 +46,11 @@ function normalizeSearchInput( input = '' ) {
 	return removeAccents( input.trim().toLowerCase() );
 }
 
-const EMPTY_ARRAY = [];
-const getCurrentValue = ( filterDefinition, currentFilter ) => {
+const EMPTY_ARRAY = [] as [];
```

Comment:
This is a very minor point here, but I'd recommend avoiding type assertions (`x as T`) as much as possible. In this case, an annotation should be sufficient:

```suggestion
const EMPTY_ARRAY: [] = [];
```

They're very similar, especially in this case, but the assertion tells the compiler we know better, whereas the annotation is more of a hint. Assertions can be dangerous:

```ts
const x: number[] = [1];

const emptyAssert = x as []; // no type error

const emptyAnnotate: [] = x; // type error
// Type 'number[]' is not assignable to type '[]'.
//  Target allows only 0 element(s) but source may have more.
```